.github/workflows/test.yml

```diff
@@ -0,0 +1,12 @@
+name: Release
+on: [push]
+jobs:
```

Comment:
missing go versions matrix

Reply:
there might be some downside when using the job as status check for PRs, in that you'll have to select them all (and change the settings everytime you change the matrix)

![Screenshot 2021-05-01 at 16 15 41](https://user-images.githubusercontent.com/291474/116785233-b6968200-aa98-11eb-873d-385f7d977354.png)

Reply:
I'll think about it